make/migrations/postgresql/0030_2.0.0_schema.up.sql

```diff
@@ -1,4 +1,8 @@
 ALTER TABLE admin_job ADD COLUMN job_parameters varchar(255) Default '';
+
+/*record the data version to decide whether the data migration should be skipped*/
+ALTER TABLE schema_migrations ADD COLUMN data_version int;
```

Comment:
Can data version use the same version of schema because data migrations will  appear with schema migration when it exist?

Reply:
Seems can't. The migrations for schema and data are two separated processes. We cannot control the migration process of schema. It's possible that the schema migration success but the data migration fails

Reply:
I got it